examples/pagination/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	jira "github.com/andygrunwald/go-jira"
+)
+
+//GetAllIssues takes a jira client and returns all issues for given JQL
+func GetAllIssues(client *jira.Client, searchString string) ([]jira.Issue, error) {
+	last := 0
+	var issues []jira.Issue = nil
+	for {
+		opt := &jira.SearchOptions{
+			MaxResults: 100,
+			StartAt:    last,
+		}
+
+		chunk, resp, err := client.Issue.Search(searchString, opt)
+		if err != nil {
+			return nil, err
+		}
+
+		total := resp.Total
+		if issues == nil {
+			issues = make([]jira.Issue, 0, total)
+		}
+		issues = append(issues, chunk...)
+		last = resp.StartAt + len(chunk)
+		if last >= total {
+			break
+		}
+	}
+	return issues, nil
+}
+
+func main() {
+	jiraClient, _ := jira.NewClient(nil, "https://issues.apache.org/jira/")
```

Comment:
the error should not be ignored here since other parts of the example have proper error handling

Reply:
Added error handling